docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
```

Comment:
Unrelated to this PR, but the fact that this syntax is supported is news to me!  Where can I follow its development?